lib/__tests__/ProjectLogsManager.test.ts

```diff
@@ -1,9 +1,19 @@
-// @ts-nocheck
```

Comment:
nit, but can you move this test into `/lib/projects/__tests__/` just to be consistent with how the other tests are organized?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1345/commits/aad6d1b774389f1cc6f1e24108b61e8cc17505f8